elasticdl_preprocessing/layers/to_ragged.py

```diff
@@ -0,0 +1,58 @@
+import tensorflow as tf
+
+_COMMA_SEP = ","
+
+
+class ToRagged(tf.keras.layers.Layer):
+    """Converts a `Tensor` or `RaggedTensor` to a `RaggedTensor`,
+    dropping ignore_value cells. If the input's dtype is string, split
+    the string elements to convert the input to `RaggedTensor` firstly.
+
+    Example (Integer):
+    ```python
+        layer = ToRagged()
+        inp = tf.constant([[1], [-1], [4]], tf.int64)
+        out = layer(inp)
+        [[1], [], [4]]
+    ```
+
+    Example (String):
+    ```python
+        ayer = ToRagged()
+        inp = tf.constant([["1,2,3"], ["4,5"], [""]])
+        out = layer(inp)
+    ```
+    The expected output is `[["1", "2", "3"], ["4", "5"], []]`
```

Comment:
For string input, the rank of output will increase 1
For int input, the rank won't be changed.
It will be confusing to the users.

Reply:
So, is the expected output for string input `[[["1", "2", "3"]], [["4", "5"]], [[]]]`?